one-pager-maker/tests/integration/redux/documentsApi.test.tsx

```diff
@@ -1,120 +1,123 @@
-            const [createDocument] = result.current;
-            createDocument({
-                uid: testUserId,
-                documentData: testDocument
-            });
-        })
-
-        waitFor(() => {
-            expect(result.current[1].isSuccess).toBe(true);
-        })
-    })
-
-    test("Firestore emulators' data should be deleted before each test", async () => {
-        const fetchDocHook = renderHook(() => useFetchDocumentsQuery({uid: testUserId}), {
-            wrapper: Wrapper
-        });
-
-        waitFor(() => {
-            expect(fetchDocHook.result.current.data?.length).toBe(0)
-        })
-    })
-
-    test("createDocument should add data to cache", async () => {
-        // 先に fetch
-        const fetchDocHook = renderHook(() => useFetchDocumentsQuery({uid: testUserId}), {
-            wrapper: Wrapper
-        });
-
-        const createDocHook = renderHook(() => useCreateDocumentMutation(), {
-            wrapper: Wrapper
-        });
-
-        act(() => {
-            const [createDocument] = createDocHook.result.current;
-            for (let i = 0; i < 3; i++) {
-                createDocument({
-                    uid: testUserId,
-                    documentData: testDocument
-                });
-            }
-        });
-
-        waitFor(() => {
-            expect(fetchDocHook.result.current.data?.length).toBe(3)
-        })
-    })
-
-    test("fetchDocuments shouldn't cache duplicated data", () => {
-        const createDocHook = renderHook(() => useCreateDocumentMutation(), {
-            wrapper: Wrapper
-        });
-
-        act(() => {
-            const [createDocument] = createDocHook.result.current;
-            for (let i = 0; i < 3; i++) {
-                createDocument({
-                    uid: testUserId,
-                    documentData: testDocument
-                });
-            }
-        });
-
-        // 後から fetch
-        const fetchDocHook = renderHook(() => useFetchDocumentsQuery({uid: testUserId}), {
-            wrapper: Wrapper
-        });
-
-        waitFor(() => {
-            expect(fetchDocHook.result.current.data?.length).toBe(3)
-        })
-    })
-})
+import { expect, test } from "vitest";
+
+test("dummy", () => {
+    expect(2 + 2).toBe(4);
+});
+
+// import {ReactNode} from "react";
+// import {Provider} from "react-redux";
+// import {beforeAll, beforeEach, describe, expect, test} from "vitest";
+// import {act, renderHook, waitFor} from "@testing-library/react";
+// import {connectFirestoreEmulator} from "firebase/firestore";
+// import {store} from "../../../src/redux/store";
+// import {DocumentForCreate} from "../../../src/entity/documentType";
+// import {db} from "../../../src/firebase";
+// import {useCreateDocumentMutation, useFetchDocumentsQuery} from "../../../src/redux/document/documentsApi";
+
+// function Wrapper(props: { children: ReactNode }) {
+//     return <Provider store={store}>{props.children}</Provider>;
+// }
```

Comment:
There are still some comments left. Do you work in the progress of test?  
Or will you make next time?

Reply:
I'm gonna make test focusing on RTK Query, which means I only check if caching works correctly. I will use those comment-out statement so I left as it is.

Reply:
OK!